### ISSUE TITLE
Safari display problem

### ISSUE THREAD

**Issue by Nevyn75** [antoniandre/vue-cal-v4#340]
Hi.

I'm facing a strange behavior.
I made a simple page in vue.js with your calendar component inside.
I've put inside 3 events by code (as default array).
If I open the page with Chrome I can see all 3 events:

![Schermata 2020-10-06 alle 16 38 45](https://user-images.githubusercontent.com/65671010/95216617-9187ac80-07f2-11eb-8df0-3f990226d546.png)

From Safari, same page, but only 1 event is displayed:

![Schermata 2020-10-06 alle 16 41 04](https://user-images.githubusercontent.com/65671010/95216839-d0b5fd80-07f2-11eb-8028-ee32bfb11660.png)

Safari: v14.0 (15610.1.28.1.9, 15610)
"vue-cal": "^3.7.0"

My code:
```
<VueCal
          today-button
          :time-from="0 * 60"
          :disable-views="['years', 'year']"
          active-view="week"
          events-on-month-view="short"
          :events="events"
          class="vuecal--blue-theme"
          locale="it"
          :no-event="false"
        >
          <template v-slot:title="{ title, view }">
            <span v-if="view.id === 'years'">Years</span>
            <!-- Using Vue Cal injected Date prototypes -->
            <span v-else-if="view.id === 'year'">{{ view.startDate.format('YYYY') }}</span>
            <span v-else-if="view.id === 'month'">{{ view.startDate.format('MMMM YYYY') }}</span>
            <span v-else-if="view.id === 'week'">w{{ view.startDate.getWeek() }} ({{ view.startDate.format('MMMM YYYY') }})</span>
            <span v-else-if="view.id === 'day'">{{ view.startDate.format('dddd D MMMM (YYYY)') }}</span>
          </template>
        </VueCal>
```
```
events: [
        {
          start: '2020-10-15 8:00',
          end: '2020-10-15 16:00',
          title: 'Turno',
          icon: '', // Custom attribute.
          content: '<i class="el-icon-s-tools" style="font-size: 24px;"></i>',
          contentFull: '', // Custom attribute.
          class: 'workshift'
        },
        {
          start: '2020-10-16 16:00',
          end: '2020-10-16 24:00',
          title: 'Turno',
          icon: '', // Custom attribute.
          content: '<i class="el-icon-s-tools" style="font-size: 24px;"></i>',
          contentFull: '', // Custom attribute.
          class: 'workshift'
        },
        {
          start: '2020-10-17 16:00',
          end: '2020-10-17 24:00',
          title: 'Turno',
          icon: '', // Custom attribute.
          content: '<i class="el-icon-s-tools" style="font-size: 24px;"></i>',
          contentFull: '', // Custom attribute.
          class: 'workshift'
        }
      ]
```

What am I missing?

Thank you.

**Comment by Nevyn75** [comment]
It seems connected to 24:00 as end time...
If I use 24:00 as end time the event is represented on Chrome but not on Safari.
If I use 00:00 of the next day as end date, the event is represented in both browsers.

**Comment by antoniandre** [comment]
Hi @Nevyn75,
Mmm good to know. thanks for reporting, I'll have a closer look soon.
I know Safari handles the date formatting differently, it needs a `T` in `2020-10-17 24:00` like `'2020-10-17T24:00'` but I have done that internally, maybe not for the particular case of end at 24.

For now, don't worry about using 00:00 the next day, Vue Cal will convert it to 23:59:59 previous day. 
If you prefer you can also use JS dates for start and end (you can make it end at 23:59:59 or 00:00 next day), that should work on all the systems.

**Comment by antoniandre** [comment]
Hi @Nevyn75,

Sorry for the delay, after checking this it was indeed a bug when setting the end of an event to '24:00'. As Safari converts this to an invalid date.

This is now fixed in version 3.8.4.

Thanks for reporting!
